### ISSUE TITLE
add defaults for ng lint

### ISSUE THREAD

**Issue by jscharett** [angular/angular-cli#7618]
[x] feature request

### Versions.
@angular/cli: 1.3.2
node: 6.9.3
os: win32 x64
@angular/animations: 4.3.6
@angular/common: 4.3.6
@angular/compiler: 4.3.6
@angular/core: 4.3.6
@angular/forms: 4.3.6
@angular/http: 4.3.6
@angular/platform-browser: 4.3.6
@angular/platform-browser-dynamic: 4.3.6
@angular/router: 4.3.6
@angular/cli: 1.3.2
@angular/compiler-cli: 4.3.6
@angular/language-service: 4.3.6

### Desired functionality.
Would be nice to set defaults for lint vs having to set them on the command line all the time.  Specifically, setting --type-check and --format would be useful, then I can just run `ng lint` and not `ng lint --format stylish --type-check`.

**Comment by filipesilva** [comment]
I agree it would be nice. We've been moving options to the cli config and this could go there as well.

**Comment by lacolaco** [comment]
I agree on this but there's one thing I concern. The description for `defaults` is specified for `generate` command.

```
    "defaults": {
      "description": "Specify the default values for generating.",
```

https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/lib/config/schema.json#L300-L301

IMO, it should be like `Specify the default options for ng commands.`. how do you think?

**Comment by lacolaco** [comment]
For backword-compatibility, my idea is a new option like `defaultOptions`.
```
  "defaultOptions": {
    "generate": { lintFix, component: {}, directive: {}, ..., },
    "lint:" { format, force, fix, ... },
  }
```
